scripts/pca_variant_filter.py

```diff
@@ -74,313 +72,369 @@ def determine_pca_variants(
+                        )
+                    ),  # Num hom ref genotypes
+                    hl.int32(
+                        (
+                            gnomad_ht.gnomad_AC
+                            - (gnomad_ht.gnomad_genomes_homozygote_count * 2)
+                        )
+                    ),  # Num het genotypes
+                    gnomad_ht.gnomad_genomes_homozygote_count,  # Num hom alt genotypes
+                ),
+            )
 
-        if min_gnomad_v3_ac:
+        ukbb_ht = hl.read_table(ukbb_release_ht_path("broad", 7))
+        ukbb_ht = ukbb_ht.select(
+            ukbb_AC=ukbb_ht.freq[0].AC, ukbb_AN=ukbb_ht.freq[0].AN,
+        )
+        ukbb_meta_ht = hl.read_table(ukbb_meta_ht_path("broad", 7))
+
+        # Only count samples used in the UK Biobank exome frequency calculations
+        ukbb_exome_count = ukbb_meta_ht.filter(
+            ukbb_meta_ht.sample_filters.high_quality
+            & hl.is_defined(ukbb_meta_ht.ukbb_meta.batch)
+            & ~ukbb_meta_ht.sample_filters.related
+        ).count()
+
+        logger.info("Getting CCDG genome and exome sample counts...")
+        ccdg_genome_count = get_ccdg_vds(
+            "genomes", filter=filter_washu
+        ).variant_data.count_cols()
+        print(ccdg_genome_count)
+        ccdg_exome_count = get_ccdg_vds("exomes").variant_data.count_cols()
+
+        def _initial_filter(data_type):
+            """
+            Get Table of CCDG variants passing desired filters.
+
+            Possible filters are:
+                - Autosomes only
+                - SNVs only
+                - gnomAD v3.1.2 AC filter
+                - CCDG high quality exome intervals
+                - UK Biobank high quality exome intervals
+
+            After densification of the VDS, rows are annotated with:
+                - ccdg_{data_type}_was_split
+                - ccdg_{data_type}_AC
+                - ccdg_{data_type}_AN
+
+            The filtered and annotated rows are returned as a Table and are also checkpointed
+            :param data_type: Whether data is from genomes or exomes
+
+            :return: Table of CCDG filtered variants
+            """
             logger.info(
-                "Filtering CCDG %s VDS to gnomAD v3.1.2 variants with adj-filtered AC > %d...",
+                "Loading CCDG %s VDS and splitting multi-allelics for initial filtering steps...",
                 data_type,
-                min_gnomad_v3_ac,
             )
-            variant_filter_expr &= gnomad_ht[ht.key].gnomad_AC > min_gnomad_v3_ac
+            vds = get_ccdg_vds(data_type, filter=filter_washu)
+            print(vds.variant_data.count_cols())
+            vds = hl.vds.split_multi(vds)
 
-        vds = hl.vds.filter_variants(vds, ht.filter(variant_filter_expr), keep=True)
+            if autosomes_only:
+                logger.info("Filtering CCDG %s VDS to autosomes...", data_type)
+                vds = filter_to_autosomes(
```

Comment:
Is something like `hl.vds.filter_to_autosomes` available now to switch to? If so, use that and remove TODO

Reply:
It's been fixed with `vds = hl.vds.filter_chromosomes(vds, keep_autosomes=True)`